package.json

```diff
@@ -15,29 +15,29 @@
   ],
   "private": true,
   "dependencies": {
-    "@angular/animations": "7.2.4",
-    "@angular/common": "7.2.4",
-    "@angular/compiler": "7.2.4",
-    "@angular/core": "7.2.4",
-    "@angular/forms": "7.2.4",
-    "@angular/platform-browser": "7.2.4",
-    "@angular/platform-browser-dynamic": "7.2.4",
-    "@angular/router": "7.2.4",
+    "@angular/animations": "^8.0.2",
+    "@angular/common": "^8.0.2",
+    "@angular/compiler": "^8.0.2",
+    "@angular/core": "^8.0.2",
+    "@angular/forms": "^8.0.2",
+    "@angular/platform-browser": "^8.0.2",
+    "@angular/platform-browser-dynamic": "^8.0.2",
+    "@angular/router": "^8.0.2",
     "core-js": "^2.4.1",
     "marked": "^0.3.9",
-    "rxjs": "^6.4.0",
+    "rxjs": "^6.5.2",
     "tslib": "^1.9.0",
-    "zone.js": "^0.8.29"
+    "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
-    "@angular/cli": "^7.3.1",
-    "@angular/compiler-cli": "7.2.4",
-    "@angular/language-service": "7.2.4",
+    "@angular-devkit/build-angular": "~0.800.0",
+    "@angular/cli": "^8.0.3",
+    "@angular/compiler-cli": "^8.0.2",
+    "@angular/language-service": "^8.0.2",
     "@types/jasmine": "~2.5.53",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "^9.4.0",
```

Comment:
If we're bumping Node support up, couldn't this also be upgraded to v10?
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/node/v10

Reply:
Done